sessions/react-custom-hooks/usemouse/src/components/Cursor.jsx

```
@@ -0,0 +1,11 @@
+import "./Cursor.css";
+
+export default function Cursor() {
+  const [x, y] = [200, 100];
+  return (
+    <div className="cursor">
+      <div className="cursor__horizontal" style={{ top: `${y}px` }}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)